app/controllers/shared/api.php

```diff
@@ -285,7 +286,16 @@
         $hasVerifiedEmail = $user->getAttribute('emailVerification', false);
         $hasVerifiedPhone = $user->getAttribute('phoneVerification', false);
         $hasVerifiedAuthenticator = TOTP::getAuthenticatorFromUser($user)?->getAttribute('verified') ?? false;
-        $hasMoreFactors = $hasVerifiedEmail || $hasVerifiedPhone || $hasVerifiedAuthenticator;
+        $webauthnAuthenticators = WebAuthn::getAuthenticatorsFromUser($user) ?? [];
```

Comment:
`Webauthn::getAuthenticatorsFromUser` does use the data structure from the user object, it's just a helper function to get all the webauthn specific ones:

<img width="782" alt="image" src="https://github.com/appwrite/appwrite/assets/26739219/7827e52f-a466-49b7-86c1-70759af291b8">

We have to do something slightly different here because this MFA WebAuthn PR has been developed with the ability to add multiple authenticators in mind. This is so users can have backup WebAuthn keys.